### PR TITLE
Updated to use Managed disks

### PR DESCRIPTION
**Pull request by varunbeesu**
Removed vhd uri to use MD
Added image references

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@varunbeesu - this is a significant change for the scenario in this template, such that we need to get CI running on it... looks like it needs a VHD uri for the test, which is now available via GEN-GENERALIZED-WINVHD-URI...  I realize this is more work than just the switch, but it pretty fundamental to the change.  Can you get a CI submission on this (i.e. remove .ci_skip from the sample)

**Comment by varunbeesu** [comment]
@bmoore-msft - I can remove .ci_skip and resubmit it, but I am not clear on VHD uri part. Do you want me to add GEN-GENERALIZED-WINVHD-URI at imageLocation -> parameters template?

**Comment by bmoore-msft** [comment]
Yes, GEN-GENERALIZED-WINVHD-URI should give you a generalize windows image that can be used

File: 301-custom-images-at-scale/final_VMSS.json [hunk 1]
```diff
@@ -43,11 +43,28 @@
...
+      "type": "Microsoft.Compute/images", 
+      "apiVersion": "2017-03-30", 
+      "name": "[variables('imageName')]", 
+      "location": "[resourceGroup().location]", 
+      "properties": { 
+        "storageProfile": { 
+          "osDisk": { 
+            "osType": "[parameters('OSType')]",
+            "osState": "Generalized", 
+            "blobUri": "[concat('https://',uniqueString(concat(resourceGroup().id, parameters('uniquePrefix'), copyIndex())), parameters('uniquePrefix'),'.blob.core.windows.net/vhds/',parameters('blobName'))]",
```
Comment (bmoore-msft) [inline 301-custom-images-at-scale/final_VMSS.json]: need to use the reference function to get the uri

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**